wagtail/documents/__init__.py

```diff
@@ -14,7 +14,7 @@ def get_document_model_string():
 def get_document_model():
     """
     Get the document model from the ``WAGTAILDOCS_DOCUMENT_MODEL`` setting.
-    Defaults to the standard :class:`~wagtail.documents.models.Document` model
+    Defaults to the standard `~wagtail.documents.models.Document` model
```

Comment:
Docstrings still use RST and not markdown, so we need double backticks. We should also remove the `~` because it's part of the `:class:` syntax to make it a shorthand, which is no longer used if we remove the `:class:`.

```suggestion
    Defaults to the standard ``wagtail.documents.models.Document`` model
```